src/theme.css

```diff
@@ -7,5 +7,6 @@
   --button-color: white;
   --section-background-color: #f1f0f0;
   --project-font-family: sans-serif;
+  --main-text-color: #666666;
```

Comment:
` --main-text-color` ->  `--primary-text-color`?
primary, secondary, standard, alert, emphasis, highlight is more common and not main